src/ensembl/webpack/environments/webpack.config.dev.js

```diff
@@ -29,14 +29,37 @@ const devServerConfig = {
     });
   },
 
-  // rules to proxy requests to the backend server in development
-  proxy: {
-    '/api': {
+  /**
+   * Rules to proxy requests to the backend server in development.
+   * The `context` field can be either a string or an array of strings for matching routes.
+   * If you want to match just a subgroup of urls within a namespace,
+   * you can add an exclusion rule to the array.
+   * Example: { context: ['/api/**', '!/api/docs/**'] }
+   * will match all routes in the `/api` namespace except for `/api/docs`.
+   *
+   * Notice that, according to the docs, the context array cannot contain a mix
+   * of string paths and wildcard paths.
+   * Valid examples:
+   *  - only string paths: { context: '/foo' }, { context: ['/foo', '/bar'] }
+   *  - only wildcard paths: { context: ['/api/**', '!/api/docs/**'] }
+   * Invalid example:
+   *  - mix of string and wildcard paths: { context: ['/api', '!/api/docs/**'] }
+   */
+  proxy: [
+    {
+      context: ['/api/**', '!/api/docs/**'],
       target: 'https://staging-2020.ensembl.org',
       changeOrigin: true,
       secure: false
+    },
+    {
+      context: ['/api/docs'],
+      target: 'http://localhost:3000',
+      pathRewrite: { '^/api/docs': '/api' },
+      changeOrigin: true,
+      secure: false
```

Comment:
**Note to self:** remember to remove this proxy rule (and update the main rule not to exclude the docs) before the merge